src/wrappers/msvc_wrapper.cpp

```diff
@@ -103,7 +109,21 @@ string_list_t make_preprocessor_cmd(const string_list_t& args) {
 }
 }  // namespace
 
-msvc_wrapper_t::msvc_wrapper_t(const string_list_t& args) : program_wrapper_t(args) {
+msvc_wrapper_t::msvc_wrapper_t(const string_list_t& args) : program_wrapper_t(m_real_args) {
```

Comment:
I am not thrilled with passing m_real_args to the parent class before it is initialized.  If the parent class ever does something with it in the constructor, behavior is undefined.  But as we are passing in a const&, I don't think there are any other good options?  Open to suggestions here.

Reply:
Have a look at  `resolve_args()` (e.g. as implemented in https://github.com/mbitsnbites/buildcache/blob/master/src/wrappers/ti_common_wrapper.cpp#L119). It's specifically there to handle response files.

Reply:
That's interesting, thank you for pointing it out.  Sorry I missed it earlier.  I like using a virtual function for this.  It is too bad that this solution means we have to ignore the parent class m_args variable, but overall it seems to be working well.  I will update to use it later tonight once I'm off work.